README.md

```diff
@@ -44,7 +44,7 @@ Simplified descriptions are available as [JSON schema](https://json-schema.org/)
 | bioimageio.spec version | JSON schema |
 | --- | --- |
 | latest | [bioimageio_schema_latest.json](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/bioimageio_schema_latest.json) |
-| 0.5 | [bioimageio_schema_v0-5.json](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/bioimageio_schema_v0-5.json) |
+| 0.5 | [bioimageio_schema_v0-5.json](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/bioimageio_schema_v0-5.json) [rendered](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/interactive_docs_v0-5.html) |
```

Comment:
the rendered dev docs are more than just documenting the json schema, as they traverse the Python source code, right?
should we link them separately then?